### PR TITLE
Made images fit inside dialog. Introduced width specifier or magnific…

### PR DESCRIPTION
**Pull request by mikkelking**
Made it a bit smoother - preserves aspect ratio if in portrait mode, and images stay inside the box. Moved the buttons above the images for smaller screens

### CODE REVIEW

File: packages/camera/package.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Package.describe({
+  name: "mikkelking:camera",
```
Comment (StorytellerCZ) [inline packages/camera/package.js:2]: I think there is a bit of a confusion on getting the package location in the repo.

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]
Looks like there is a confusion on the location of the files in the repository.

File: packages/camera/package.js (hunk 1 above)
Comment (StorytellerCZ) [reply packages/camera/package.js:2]: Yes, both the package name and package should be `mdg:camera`.

**Comment by StorytellerCZ** [comment]
This PR is malformed. The folder for the camera package is wrong which means that it looks like you are just creating a new package. It should be in `mdg:camera` folder. Please fix, so that we can proceed.

**Comment by filipenevola** [comment]
It's probably easier to open a new PR.

**Pull request closed without merging**